test/largescale/space/fn_base_slow_test.py

```diff
@@ -0,0 +1,306 @@
+# Copyright 2014, 2015 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import range
+
+# External module imports
+import pytest
+import numpy as np
+
+# ODL imports
+import odl
+
+from odl.util.testutils import all_almost_equal, almost_equal
+
+pytestmark = pytest.mark.skipif(
+    "not pytest.config.getoption('--largescale')",
+    reason='Need --largescale option to run'
+)
+
```

Comment:
I like this option thing. In fact, I think it's so useful that this decorator (along with its benchmark counterpart) should be put into `testutils`.